src/backend/dx11/Cargo.toml

```diff
@@ -30,6 +30,6 @@ name = "gfx_device_dx11"
 log = "0.3"
 gfx_core = { path = "../../core", version = "0.2" }
 d3d11-sys = "0.1"
```

Comment:
Sorry, I missed it. We'll update this at any point later, since it's not necessary and doesn't break the API.